service/graph/nodes/step_node.py

```
@@ -0,0 +1,71 @@
+        {"role": "system", "content": _STEP_SYSTEM_PROMPT},
+        {"role": "user", "content": f"현재 단계: {current_step}\n사용자 발화: {last_user_msg}"},
+    ])
+
+    raw = response.content.strip()
+    next_step = None if raw.lower() == "null" else raw
+
+    if next_step and session_id:
+        spring = get_spring_adapter()
+        try:
+            await update_step(spring, session_id, next_step)
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)